### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#310] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322287-dc875dcf-1d3f-46de-a21d-4789884aa837.png)

**Comment by Exyss** [comment]
![Esercizio 10](https://user-images.githubusercontent.com/65043207/152646802-a7f8583f-66e3-4128-a7bc-b547f50022fd.jpg)

**Comment by Elia-Belli** [comment]
Confermo la soluzione di @Exyss 
- **Caso Base**: $n=1$ , $1 \times 2 = (1-1)\times 2^2 + 2 \rightarrow 2 = (0\times 4) +2$
- **Ipotesi Induttiva**: $P(n) = \sum\limits_{i=1}^{n} i\times 2^i = (n-1)\times 2^{n+1} + 2$
- **Passo Induttivo**: $P(n) \Rightarrow P(n+1) = \sum\limits_{i=1}^{n+1} i\times 2^i = n\times {2^{n+2}} + 2$
$\sum\limits_{i=1}^{n} i\times 2^i + (n+1)\times {2^{n+1}}= n\times {2^{n+2}} + 2$ applico $P(n)$
$(n-1)\times 2^{n+1} + 2 + (n+1)\times {2^{n+1}}= n\times {2^{n+2}} + 2$
$2^{n+1}\times (n-1+n+1) =  n\times {2^{n+2}}$
$2^{n+1}\times (2n) = n\times {2^{n+2}}$
$2^{n+2}\times n = n\times {2^{n+2}}$
